### PR TITLE
Ui Changes and Added functionality of designation

### PR DESCRIPTION
**Pull request by TusharGupta03**

### CODE REVIEW

**Comment by TusharGupta03** [comment]
@varun7singh

**Comment by TusharGupta03** [comment]
Updated sidebar , app bar , dashboard ,added functionality of selecting multiple designation through dropdown in appBar

**Comment by TusharGupta03** [comment]
@Shivkant-Chauhan

**Comment by TusharGupta03** [comment]
@ChaudharyRaman

File: android/build.gradle [hunk 1]
```diff
@@ -1,12 +1,12 @@
 buildscript {
-    ext.kotlin_version = '1.6.10'
+    ext.kotlin_version = '1.8.21'
```
Comment (Shivkant-Chauhan) [inline android/build.gradle:2]: did you confirm this changing of kotlin version with the Android leads (Priyanshu or Pranjal)

File: android/build.gradle [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
     repositories {
         google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.2'
+        classpath 'com.android.tools.build:gradle:7.3.0'
```
Comment (Shivkant-Chauhan) [inline android/build.gradle:9]: Here as well

**Review by Shivkant-Chauhan — CHANGES_REQUESTED** [review]
Unable to review fully, GitHub leads will take care of it

File: android/build.gradle (hunk 1 above)
Comment (TusharGupta03) [reply android/build.gradle:2]: Yes

File: android/build.gradle (hunk 2 above)
Comment (TusharGupta03) [reply android/build.gradle]: We have updated the gradle version after confirming leads to 7.4.2

**Comment by PraneGIT** [comment]
LGTM. Merging this.

**Pull request merged**